src/components/common/navigation/Navigation.tsx

```diff
@@ -278,11 +281,13 @@ export default class Navigation extends Component<
                         </NavLink>
                         {NavigationList.map((item, index) => {
                             if (
-                                (!item.forceHideEnvKey ||
-                                    (item.forceHideEnvKey && !window?._env_?.[item.forceHideEnvKey])) &&
-                                ((this.props.serverMode !== SERVER_MODE.EA_ONLY && !item.moduleName) ||
-                                    (this.props.serverMode === SERVER_MODE.EA_ONLY && item.isAvailableInEA) ||
-                                    this.props.installedModuleMap.current?.[item.moduleName])
+                                (!window._env_.K8S_CLIENT &&
+                                    (!item.forceHideEnvKey ||
+                                        (item.forceHideEnvKey && !window?._env_?.[item.forceHideEnvKey])) &&
```

Comment:
remove these existing optional checks or add to line 284 if required

Reply:
Done